doc/legacy_mode/main.md

```diff
@@ -36,20 +36,23 @@ You may also configure Legacy Mode via configuration parameters.
 
 ### Show Output
 
-The `"rust.showOutput"` configuration parameter controls whether the Output channel should be shown when [a Cargo command starts executing](../cargo_command_execution.md). All Cargo command other than `build` and `check` trigger this event.
+The `"rust.showOutput"` configuration parameter controls whether the output channel should be shown when [a Cargo command starts executing](../cargo_command_execution.md).
 
 The possible values:
 
 * `true` - the output channel should be shown
 * `false` - the output channel shouldn't be shown
 
-When `"rust.executeCargoCommandInTerminal"` is set to `true`, `"rust.showOutput"` has no effect.
+The output channel will not be shown under the following conditions:

```

Comment:
For me, as not native speaker, it is ambiguous if the output channel will not be shown if all the conditions are met or at least one.